supplementary_style_guide/style_guidelines/managed-services.adoc

```diff
@@ -95,7 +95,7 @@ See link:https://www.patternfly.org/v4/ux-writing/about[UX writing] in the Patte
 
 [[screenshots]]
 == Screenshots
-Avoid screenshots for both accessibility and localization reasons. If you must use screenshots use them as judiciously as possible and ensure alt text is unique and descriptive. See xref:accessibility[Accessibility] for more information on proper use of images in user interface documentation.
+Avoid screenshots for both accessibility and localization reasons. If you must use screenshots use them as judiciously as possible and ensure alt text is unique and descriptive. For more information on proper use of images in user interface documentation, see xref:accessibility[Accessibility].
```

Comment:
If you change the previous 'information on' to information about' you might as well do this one too.